tests/pytest/test_multi.py

```diff
@@ -663,7 +666,7 @@ def testObjLenCommand(env):
     r.assertOk(r.execute_command('JSON.SET', 'doc1', '$', '{"nested1": {"a": {"foo": 10, "bar": 20}}, "a":["foo"], "nested2": {"a": {"baz":50}}}'))
     # Test multi
     res = r.execute_command('JSON.OBJLEN', 'doc1', '$..a')
-    r.assertEqual(res, [2, None, 1])
+    r.assertEqual(set(res), set([2, None, 1]))
```

Comment:
Same comment on whether a change of order is considered a breaking change

Reply:
I do not think it is considered a breaking change

Reply:
@gkorland @K-Jo please approve/disapprove.

Reply:
The PR returns `[None, 2, 1]` which is also what is been returned here: https://jsonpath.com/

Reply:
@gkorland agrees its not a breaking change.